pkg/engine/plan/plan.go

```diff
@@ -411,6 +411,79 @@ func (v *Visitor) AllowVisitor(kind astvisitor.VisitorKind, ref int, visitor int
 		return true
 	}
 	for _, config := range v.planners {
+		// In the following query:
```

Comment:
@csilvers given your recent experience with planner paths, is this comment any clearer, or do you think it could still benefit from some additional context?

Reply:
It's definitely clearer, but I'm still not sure it's clear!  But my confusion runs pretty deep, and it's probably best to discuss in person if you want to make any more changes.  (To be clear, there's no need to make changes on my account, if jens thinks this is clear.)